crates/control/src/ball_filter.rs

```diff
@@ -95,16 +97,25 @@ impl BallFilter {
         &mut self,
         measurements: Vec<(&SystemTime, Vec<&Ball>)>,
         context: &CycleContext,
-    ) -> Vec<Hypothesis> {
+    ) -> RemovedHypotheses {
+        let cycle_time = Duration::from_secs_f32(0.012);
+        let filter_parameters = &context.ball_filter_configuration;
+        let velocity_threshold = filter_parameters.measurement_matching_distance;
```

Comment:
huh?!

Reply:
Not sure what you mean. Removed `velocity_threshold` variable for now

Reply:
The `velocity_threshold` is assigned with a value called `measurement_matching_distance`?!?!?

Reply:
hehe, not sure how that happened, fixed